src/core_init_atmosphere/mpas_init_atm_gwd.F

```diff
@@ -360,66 +552,54 @@ function read_global_30s_topo(path, sub_path) result(iErr)
       ny = tile_y + 2*tile_bdr
       nz = 1
 
-      ishift = 0
-
-      !
-      ! For GMTED2010 data, the dataset starts at 0.0 longitude, but we need to shift the starting location
-      ! in the topo array to -180.0, so we introduce an offset in the x-coordinate of topo_x/2
-      !
-      if (trim(sub_path) == 'topo_gmted2010_30s/') then
-         ishift = topo_x / 2
+      iy = tile_start_y
+      ix = tile_start_x
+
+      !write(filename,'(a,i5.5,a1,i5.5,a1,i5.5,a1,i5.5)') trim(path)//trim(sub_path), tile_start_x, '-', (tile_start_x+tile_x-1), '.', &
+      !tile_start_y, '-', (tile_start_y+tile_y-1)
```

Comment:
Are these commented-out lines likely to be needed in future?

Reply:
Removed it.